tools/configurator/configurator.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
```

Comment:
need a string here for the description arg on L19

Reply:
fixed